-------------------------------------------------------------------------------
Test set: com.utilidades.S1Test
-------------------------------------------------------------------------------
Tests run: 4, Failures: 2, Errors: 0, Skipped: 0, Time elapsed: 0.049 sec <<< FAILURE!
com.utilidades.S1Test.testPagarEntrada()  Time elapsed: 0.033 sec  <<< FAILURE!
org.opentest4j.MultipleFailuresError: Multiple Failures (6 failures)
	org.opentest4j.AssertionFailedError: Edad 5: falso ==> expected: <false> but was: <true>
	org.opentest4j.AssertionFailedError: Edad 18: verdadero ==> expected: <true> but was: <false>
	org.opentest4j.AssertionFailedError: Edad 41: verdadero ==> expected: <true> but was: <false>
	org.opentest4j.AssertionFailedError: Edad 65: falso ==> expected: <false> but was: <true>
	org.opentest4j.AssertionFailedError: Edad 80: falso ==> expected: <false> but was: <true>
	org.opentest4j.AssertionFailedError: Edad -1: nulo ==> expected: <null> but was: <true>
	at org.junit.jupiter.api.AssertAll.assertAll(AssertAll.java:80)
	at org.junit.jupiter.api.AssertAll.assertAll(AssertAll.java:44)
	at org.junit.jupiter.api.AssertAll.assertAll(AssertAll.java:38)
	at org.junit.jupiter.api.Assertions.assertAll(Assertions.java:2911)
	at com.utilidades.S1Test.testPagarEntrada(S1Test.java:54)

com.utilidades.S1Test.testEntreCeroYDiez()  Time elapsed: 0 sec  <<< FAILURE!
org.opentest4j.MultipleFailuresError: Multiple Failures (2 failures)
	org.opentest4j.AssertionFailedError: Valor -7: falso ==> expected: <false> but was: <true>
	org.opentest4j.AssertionFailedError: Valor 34: falso ==> expected: <false> but was: <true>
	at org.junit.jupiter.api.AssertAll.assertAll(AssertAll.java:80)
	at org.junit.jupiter.api.AssertAll.assertAll(AssertAll.java:44)
	at org.junit.jupiter.api.AssertAll.assertAll(AssertAll.java:38)
	at org.junit.jupiter.api.Assertions.assertAll(Assertions.java:2911)
	at com.utilidades.S1Test.testEntreCeroYDiez(S1Test.java:37)

